lib/tasks/acceptance_tests.rake

```diff
@@ -0,0 +1,7 @@
+namespace :acceptance_tests do
+  desc 'Run acceptance tests'
+  task :run, [:url] do |task, args|
+    #Example command: bundle exec rake acceptance_tests:run[verify-self-service-dev.cloudapps.digital]
+    sh "TEST_URL=#{args[:url]} bundle exec rspec spec/system/acceptance --tag acceptance"
```

Comment:
You can use [RSpec's rake take](https://relishapp.com/rspec/rspec-core/docs/command-line/rake-task) here so you don't have to shell out.